Update tutorial to 5.x version of sdk

joliveros opened the pull request.

acamino committed:
Add tutorial

acamino committed:
Update tutorial steps

acamino committed:
Edit tutorial

kwhinnery committed:
Update README.md

joliveros committed:
Update twilio-csharp to 5.0.0-rca2

joliveros committed:
Update controllers to use updated twilio sdk

joliveros committed:
Update tests

joliveros committed:
Add default AccountSid and authToken values to configuration

joliveros committed:
Update readme

joliveros committed:
Delete unnecessary tests

joliveros committed:
Update MessageSender with better client init, minor code style changes

joliveros committed:
Update tutorial to twilio-csharp 5.x

acamino commented:
Thanks @joliveros.

- In the section **Creating a Twilio REST API Client** I would highlight the code from line 9 up to line 12. What about removing the paragraph "Credentials and PhoneNumbers ..." and the code example about the `AccountSID` property, IMO that one is just noise.
- In the section **Sending the Message**. Should it be `Create` or `CreateAsync`?

Would you please take care of the requested changes?

joliveros committed:
Merge branch 'gh-pages' into gh-pages-next-gen

acamino committed:
Refactor tests to make them more explicit

acamino committed:
Refactor `Configuration.cs` to use expression body

acamino committed:
User string instead of String

acamino committed:
Refactor: Rename MessageSender to NotificationService

Update tests to gain confidence. Usually it's better to write an assertion
against to a concrete value instead of "any" string.

acamino committed:
Remove `ace` folder

acamino committed:
Update highlighting on the tutorial

acamino commented:
@joliveros please take a look at this one.

acamino committed:
Update README to include `Local.config.example` instructions

joliveros commented:
LGTM @acamino, thank you!
